folsom/src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -223,8 +223,11 @@ private DefaultRawMemcacheClient(
       SetRequest setRequest = (SetRequest) request;
       byte[] value = setRequest.getValue();
       if (value.length > maxSetLength) {
-        return (CompletionStage<T>)
-            onExecutor(CompletableFuture.completedFuture(MemcacheStatus.VALUE_TOO_LARGE));
+        final CompletionStage<T> future =
+            (CompletionStage<T>)
+                onExecutor(CompletableFuture.completedFuture(MemcacheStatus.VALUE_TOO_LARGE));
+        metrics.measureFuture(future, address.getHostText());
```

Comment:
that's not an interesting case to measure per host. What about other futures - they are not measured?

Reply:
What do you mean by saying other futures ?
All rawMemcacheClient's using the send method and we wanted to measure globaly and not per action Get,Set and etc..